src/appleseed/renderer/kernel/lighting/lightsamplerbase.cpp

```diff
@@ -355,7 +355,8 @@ void LightSamplerBase::collect_emitting_shapes(
             o = global_transform.point_to_parent(o);
             x = global_transform.vector_to_parent(x);
             y = global_transform.vector_to_parent(y);
-            n = global_transform.normal_to_parent(n);
+            n = normalize(global_transform.normal_to_parent(n));
+            assert(is_normalized(n));

```

Comment:
I suppose the assert is unnecessary here.

Reply:
Yep it is :)